data/selinux/COPYING

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
I wonder if this should go to top-level as COPYING.GPL2, @mvo5 what do you think.

We should be clear about the license that applies to particular part of the code.

@Conan-Kudo I think we need to document the license in the particular files to be compliant.